generator/final_processor_country.cpp

```diff
@@ -322,6 +321,7 @@ void CountryFinalProcessor::DropProhibitedSpeedCameras()
     ForEachFeatureRawFormat<serialization_policy::MaxAccuracy>(path, [&](FeatureBuilder const & fb, uint64_t)
     {
       // Removing point features with speed cameras type from geometry index for some countries.
+      auto const speedCameraType = classif().GetTypeByPath({"highway", "speed_camera"});
```

Comment:
No, put it without static on the initial place, to calculate only once.

Reply:
done